downstream/modules/platform/proc-add-eda-safe-plugin-var.adoc

```diff
@@ -0,0 +1,25 @@
+
+[id="proc-add-eda-safe-plugin-var"]
+
+= Adding a safe plugin variable to {EDAcontroller}
+
+When using redhat.insights_eda or similar plug-ins to run rulebook activations in {EDAcontroller}, you must add a safe plugin variable to a directory in {PlatformNameShort}. This would ensure connecton between {EDAcontroller} and the source plugin, and display port mappings correctly. 
```

Comment:
Style suggests avoiding use of the term "would" unless referring to an unfulfilled condition. Suggest rewriting to either:
This ensures ...
or
This will ensure ...

Reply:
Thanks, @dcdacosta. Will handle in a separate PR..